### ISSUE TITLE
Proposal: Drop support for Ruby 1.8.7, maybe upto 2.3

### ISSUE THREAD

**Issue by mjy** [nexml/bio-nexml#11]
If it was up to me I'd drop all the way to 2.3 or so.  Thoughts?

**Comment by yeban** [comment]
I agree to drop support till Ruby 2.0. I still see may Ruby 2.1.5 installations. Nokogiri (wrt #7) also supports Ruby 2.1 still. Ruby 2.1 captures most syntactic improvement we have grown used to (symbol hash syntax, keyword arguments, lambda operator). In addition, refinements API and module#prepend that were finalised in 2.1 are helpful.
